api/v1alpha1/databaseclusterbackup_types.go

```diff
@@ -71,6 +74,13 @@ type DatabaseClusterBackupList struct {
 	Items           []DatabaseClusterBackup `json:"items"`
 }
 
+// IsComplete returns true if the backup has completed.
+func (b DatabaseClusterBackup) IsComplete() bool {
+	return b.Status.State == BackupState(pxcv1.BackupSucceeded) ||
+		b.Status.State == BackupState(pgv2.BackupSucceeded) ||
+		b.Status.State == BackupState(psmdbv1.BackupStateReady)
```

Comment:
What about the failed states? I think they are also terminal states.

Reply:
right, thanks. Updated